src/sentry/static/sentry/app/components/acl/feature.jsx

```diff
@@ -102,12 +102,18 @@ class Feature extends React.Component {
   }
 
   hasFeature(feature, features) {
-    let shouldMatchOnlyProject = feature.match(/^project:(\w+)/);
-    let shouldMatchOnlyOrg = feature.match(/^organization:(\w+)/);
+    let shouldMatchOnlyProject = feature.match(/^projects:(.+)/);
+    let shouldMatchOnlyOrg = feature.match(/^organizations:(.+)/);
 
     // Array of feature strings
     let {configFeatures, organization, project} = features;
 
+    // Check config store first as this ovverides features scopped to org or
+    // project contexts.
+    if (configFeatures.includes(feature)) {
```

Comment:
Is there a test for this behavior?

Reply:
Yes, this one started failing when I changed the prefix from organization to organizations

https://github.com/getsentry/sentry/blob/d9555a56cfc361fefe73ee509b36871ba7c29e57/tests/js/spec/components/acl/feature.spec.jsx#L183-L200